Assignment2.yaml

```diff
@@ -0,0 +1,415 @@
+  /species:
+    get:
+      tags:
+      - "Species"
+      summary: "Find list of species"
+      description: "get all the species resources"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/species"
+  /species/{speciesId}:
+    get:
+      tags:
+      - "Species"
+      summary: "Find speciese by ID"
+      description: "get a specific species resource"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "speciesId"
+        in: "path"
+        description: "ID of species to return"
+        required: true
+        type: "number"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/species"
+  /species/schema:
+    get:
+      tags:
+      - "Species"
+      summary: "Return Schema"
+      description: "Returns a json Schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/species"
+definitions:
+  Vehicle:
+    type: "object"
+    required:
+    - "name"
+    properties:
+      model:
+        type: "string"
+        example: "All-Terrain Attack Transport"
+      Vehicle_Class:
+        type: "string"
+      name:
+        type: "string"
+        example: "speeder bike"
+      Manufacturer:
+        type: "string"
+      Length:
+        type: "integer"
+        format: "int32"
+      cost_in_credits:
+        type: "string"
+      max_atmospheric_speed:
+        type: "string"
+      cargo-capacity:
+        type: "string"
+      consumables:
+        type: "string"
+      films:
+        type: "string"
+      pilots:
+        type: "string"
+      url:
+        type: "string"
+      credited:
+        type: "string"
+      edited:
+        type: "string"
+  people:
+    type: "object"
+    required:
+    - "name"
+    properties:
+      name:
+        type: "string"
+        example: "Shiv"
+      birth_year:
+        type: "string"
+      eye_color:
+        type: "string"
+      gender:
+        type: "string"
+        example: "brown"
+      hair_color:
+        type: "string"
+        example: "black"
+      height:
+        type: "string"
+      mass:
+        type: "string"
+      skin_color:
+        type: "string"
+      homeworld:
+        type: "string"
+      films:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/planets/1/]"
+      species:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/species/1/]"
+      starships:
+        type: "array"
+        items:
+          type: "string"
+        example: "[http://swapi.co/api/starships/12/]"
```

Comment:
Is this `array` or a `string`?
There is a difference between "[http://swapi.co/api/starships/12/]" and [ "http://swapi.co/api/starships/12/" ]